frontend/ui/elements/screensaver_menu.lua

```diff
@@ -176,3 +186,11 @@ return {
         },
     },
 }
+
+if Device:isTouchDevice() then
+    table.insert(menu_items[1].sub_item_table[8].sub_item_table,
```

Comment:
Possibly `#menu_items[1].sub_item_table` instead of `8`? (Assuming it's indeed the final element).

Or anything else you might come up with to avoid the magic number, basically ;).

Reply:
iterating through all the elements and matching the name of the sub_item_table ("Postpone screen update after wake-up")?

Reply:
Can't you just on line 97:
```diff
- -- "Until 'exit sleep screen' gesture" is added later to touch devices
+ Device:isTouchDevice() and genMenuItem(_("Until 'exit sleep screen' gesture"), "screensaver_delay", "gesture") or nil
```